### PR TITLE
feat(h3framework): Added support for h3 framework

### PR DESCRIPTION
**Pull request by Kall7**
## Summary of change

Added full support for h3 framework, which is used in Nuxt3.

## Related issues

-   [h3 Support](https://github.com/supertokens/supertokens-node/issues/296)

## Test Plan

Manual testing of all functions that got added.

## Documentation changes

TODO 

## Checklist for important updates

-   [x] Changelog has been updated
-   [ ] `coreDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `lib/ts/version.ts`
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [ ] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [x] Had run `npm run build-pretty`
-   [ ] Had installed and ran the pre-commit hook
-   [x] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [x] If have added a new web framework, update the `add-ts-no-check.js` file to include that
-   [ ] If added a new recipe / api interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).
-   [ ] If added a new recipe, then make sure to expose it inside the recipe folder present in the root of this repo. We also need to expose its types.

## Remaining TODOs for this PR

-   [ ] Create tests for new framework (only made one because I couldn't run tests on windows)

### CODE REVIEW

**Comment by rishabhpoddar** [comment]
Hey @Kall7 can you please make sure that all the PR checks above pass?

**Comment by Kall7** [comment]
I'll do it this weekend

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [9.2.1] - 2022-05-05
+
+# Adds
+
+- Support for h3 freamwork.
```
Comment (rishabhpoddar) [inline CHANGELOG.md:14]: this should go under the unreleased section

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Kall7) [reply CHANGELOG.md:14]: Ah my bad. I'll fix it

Comment (Kall7) [reply CHANGELOG.md:14]: wait it is under unreleased

Comment (rishabhpoddar) [reply CHANGELOG.md:14]: It seems to be under `## [9.2.1] - 2022-05-05`

Comment (Kall7) [reply CHANGELOG.md:14]: AHHHH mb mb

Comment (Kall7) [reply CHANGELOG.md]: there

**Comment by Swiftzerr** [comment]
What is the current status on this? It seems abandoned

**Comment by rishabhpoddar** [comment]
Closing this PR since it's abandoned. Instead, continuing it here: https://github.com/supertokens/supertokens-node/pull/351.

Thanks @Kall7 for the work so far, @Swiftzerr you may want to look at the new PR and considering forking and contributing to that.

**Pull request closed without merging**